scout/server/blueprints/institutes/views.py

```diff
@@ -89,3 +90,64 @@ def institute(institute_id):
     return render_template(
         "/overview/institute.html", form=form, default_phenotypes=default_phenotypes, **data
     )
+
+
+@blueprint.route("/<institute_id>/clinvar_submissions", methods=["GET", "POST"])
+@templated("overview/clinvar_submissions.html")
+def clinvar_submissions(institute_id):
+    """Handle clinVar submission objects and files"""
+
+    def generate_csv(header, lines):
+        yield header + "\n"
+        for line in lines:  # lines have already quoted fields
+            yield line + "\n"
+
+    if request.method == "POST":
+        submission_id = request.form.get("submission_id")
```

Comment:
No default for submission_id - ok w None in the following functions (or won't happen?)

Reply:
Solved by refactoring